content/use-cases/contribute-code-to-rippled.md

```diff
@@ -0,0 +1,56 @@
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## <a href="https://github.com/ripple/rippled" target="_blank">Access the `rippled` repo</a> <i class="fa fa-external-link" aria-hidden="true"></i><!--#{ fix md highlighting_ #}-->
+
+`rippled` is an open-source project. You can take a look at `rippled` code simply by accessing the `rippled` GitHub repo. Before contributing or reporting bugs, we recommend that you get to know the code and developer experience by performing the following tasks.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up and run a `rippled` server](xxxxx.html)
+
+Set up and run a `rippled` server to understand the developer experience and functionality of the core peer-to-peer server that manages the XRP Ledger. Anyone can run their own `rippled` server that follows the network and keeps a complete copy of the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Try out XRP Ledger integration tools](xxxxx.html)
+
+Take a look at the various tools provided to help developers integrate with the XRP Ledger. From WebSocket and JSON-RPC API endpoints to the `ripple-lib` JavaScript library, take a look at the modes of integration offered to the developer community.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Get a test XRP Ledger account and test XRP](xxxxx.html)
+
+Use the XRP Ledger Test Net to get a test account and test XRP. Connect your `rippled server` to the Test Net to make test calls and get to know the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up your developer environment](xxxxx.html)
+
+<!-- TODO: awaiting links to a few rippled repo md files -->
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## <a href="https://github.com/ripple/rippled/pulls" target="_blank">Contribute code</a> <i class="fa fa-external-link" aria-hidden="true"></i><!--#{ fix md highlighting_ #}-->
+
+Now that you have a handle on `rippled`, you may have ideas for how to improve it. Perhaps you’re developing on the XRP Ledger and want to contribute some code that enables the XRP Ledger to provide a feature your application needs. Access the `rippled` repo and open an issue or pull request.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## <a href="https://github.com/ripple/rippled/issues" target="_blank">Report bugs</a> <i class="fa fa-external-link" aria-hidden="true"></i><!--#{ fix md highlighting_ #}-->
+
+As you explore rippled, you may find code that you don’t think is working as intended. To report a bug, open an issue in the rippled repo.
```

Comment:
code-font, `rippled`

Reply:
Also you can make "open an issue" an external link to https://github.com/ripple/rippled/issues (I intentionally didn't make it a link to the "New Issue" page because this way people might see that someone has already opened the same issue)